index.js

```diff
@@ -16,6 +16,10 @@ const TEN_MEGABYTES = 1000 * 1000 * 10;
 function handleArgs(cmd, args, opts) {
 	let parsed;
 
+	if (opts && opts.env && opts.extendEnv !== false) {
+		opts.env = Object.assign(process.env, opts.env);
```

Comment:
When looking back at the code, this way we extend `process.env` itself as well. Might be better to do `Object.assign({}, process.env, opts.env)` to make sure we don't alter external things.